odrpc.py

```diff
@@ -21,6 +21,9 @@ class DetectRequest:
     detector_name: Optional[str] = None
     image: Optional[str] = ""
     throttle: Optional[float] = 0.0
+    separate_detections: Optional[bool] = False
```

Comment:
For now, would you mind moving these to an `mqtt` sub class? None of these are really relevant to the API. Binary doesn't really work in JSON and crop won't really work unless we return them as part of each detection.

Reply:
Sure thing!